packages/headless-components/services/src/react/ServiceListSort.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useService } from '@wix/services-manager-react';
+import { ServicesListServiceDefinition } from '../services/services-list-service.js';
+
+export interface ServiceListSortProps {
+  children: (props: {
+    currentSort: { fieldName: string; order: 'ASC' | 'DESC' }[];
+    sortOptions: Array<{
+      fieldName: string;
+      order: 'ASC' | 'DESC';
+      label: string;
+    }>;
```

Comment:
Can we import `import { Sort as SortPrimitive } from '@wix/headless-components/react';` and try using `SortPrimitive`? Let's extend the interface if necessary